code/game/objects/structures/fireplace.dm

```diff
@@ -1,6 +1,6 @@
-#define LOG_BURN_TIMER 150
-#define PAPER_BURN_TIMER 5
-#define MAXIMUM_BURN_TIMER 3000
+#define LOG_BURN_TIMER 300
```

Comment:
why

Reply:
Ah, for testing, throwing an entire bin of paper and several logs worth of wood into the fireplace gave me about 30 seconds of light so I was messing with the numbers. Piece of shit doesn't even give off light when it has 60 seconds of flame left.